changelog.d/20241025_181736_faraz.maqsood_enable_elastic_search_for_discovery.md

```diff
@@ -0,0 +1 @@
+- [Feature] Add `ElasticSearch` support in tutor-discovery as tutor core is shifted to `MeiliSearch`. (by @Faraz32123)
```

Comment:
We can link core PR here and maybe tag this as a breaking change?

Reply:
No need to wrap Elasticsearch and Meilisearch in backticks (and there is no capital S)